.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,50 @@
+name: E2E over minikube
+
+on:
+  push:
```

Comment:
We are restricting the push event to `main` in other pipelines, which should alleviate the load while working on private branches.

Reply:
ah yes.. I using it for tests and forgot to change it.